android/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         minSdkVersion = 21
         compileSdkVersion = 31
         targetSdkVersion = 31
-        kotlinVersion = '1.6.10'
+        kotlinVersion = '1.8.0'
```

Comment:
We should look into this. I thought a lot more work was required for compatibility.

Reply:
It appears to be so - the build is still failing, and with a bunch of new errors now.